src/Umbraco.Cms.Integrations.Search.Algolia/Client/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "algolia",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=20.9 <21",
+    "npm": ">=10.1 < 11"
+  },
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "watch": "vite build --watch",
+    "generate:api": "openapi-ts --input https://localhost:44340/umbraco/swagger/AlgoliaSearch/swagger.json --output src/generated --debug true --enums typescript --lint true --schemas false"
+  },
+  "dependencies": {
+    "lit": "^3.1.2"
```

Comment:
Not sure if we need this?  See [the Forms version of this file](https://github.com/umbraco/Forms/blob/v14/dev/src/Umbraco.Forms.StaticAssets/package.json) for comparison.

Reply:
Yup, we don't need to take a direct dependency on Lit - the backoffice package provides it as an export 👍

Reply:
Removed dependency.